core/common/src/test/scala/net/liftweb/common/BoxSpec.scala

```diff
@@ -152,14 +152,17 @@ class BoxSpec extends Specification with ScalaCheck with BoxGenerator {
         Full(Empty).flatten must_== Empty
       }
     }
+    "define a 'mapFailure' method returning itself." in {
+      Full(1).mapFailure(identity) must_==  Full(1)
```

Comment:
Even if `mapFailure` didn't map just over a failure, this would return `Full(1)`. How about a parameter that returns some standard `Failure`?